doc/cprover-manual/index.md

```diff
@@ -1,34 +1,34 @@
 # The CPROVER Manual
 
-## 1. [Introduction](introduction/)
+1. [Introduction](introduction/)
 
-## 2. [Installation](installation/)
+2. [Installation](installation/)
```

Comment:
@kroening Will this render ok on cprover.org?

Reply:
FWIW, I did mention this to @kroening  (replacing the sequence of headers with a list) and he did not object.